queue/driver_group_mongo.go

```diff
@@ -184,6 +184,8 @@ func (d *mongoGroupDriver) Get(ctx context.Context, name string) (amboy.Job, err
 		return nil, errors.Wrapf(err, "GET problem fetching '%s'", name)
 	}
 
+	j.Name = j.Name[len(d.group)+1:]
```

Comment:
I don't think you want `+1` here https://play.golang.org/p/Mkm5nRuBIms

Reply:
they're dot seperated, so I want to get the dot too.